lib/api/migrations/migration-1.0.1.js

```diff
@@ -25,7 +25,7 @@ function transformRequestParamsToString(mockedRequests) {
 
       // cancel previous default value
       if (mockedRequest.params === '{}') {
-        mockedRequest.params = null;
+        mockedRequest.params = undefined;
```

Comment:
Why are you changing a migration file? Is there a bug in the migration?
If you want to make changes to the data structure you must create a new migration file that applies the changes you wish to do.